apps/passport-server/test/tripsha/mockTripshaAPI.ts

```diff
@@ -0,0 +1,78 @@
+import { RestHandler, rest } from "msw";
+import { SetupServer, setupServer } from "msw/node";
+
+export const MOCK_ZUCONNECT_TRIPSHA_URL = "http://tripsha.example.com";
+
+// An empty response
+export const badEmptyResponse = {};
+
+// A response with a faulty ticket.
+export const badTicketsResponse = {
+  tickets: [
+    {
+      id: "65012572b5f4a42c66422b9f",
+      type: "ZuConnect Resident Pass",
+      // Ticket has no email address
+      first: "Test",
+      last: "One"
```

Comment:
i wonder if their data may have missing last names or first names - this has come up as a bug during the initial implementation of the original pretix sync we made for zuzalu.